vignettes/Data_Normalization.Rmd

```diff
@@ -0,0 +1,385 @@
+                               refpool_cname = "Virus",
+                               refpool_notation = "Pool")
+
+# look at boxplots of the data after normalization to the reference pool samples
+plot(isobaric_object_normalized)
+```
+
+2. Specify the `channel_cname` and `refpool_channel`. This should be used when the reference sample for each plex/plate/experiment was always located in the same channel. Here `channel_cname` gives the column name for the column in `f_data` which gives information about which channel each sample was run on, and `refpool_channel` is a character string specifying the value in `channel_colname` that corresponds to the reference sample channel. 
+
+```{r specify_channels, eval = FALSE}
+# not run because this data does not actually contain channel_cname or refpool_channel information
+iso_norm <- normalize_isobaric(isobaric_object, 
+                               exp_cname = "Plex",
+                               apply_norm = FALSE,
+                               channel_cname = "SampleChannel", # this column in f_data would specify a number corresponding to the channel for each sample
+                               refpool_channel = 4) # this value in the SampleChannel column would always correspond to a reference pool sample
+```
+
+
+### NMR
+
+In order to ensure that samples are comparable to one another, normalizing NMR data to either a sample-specific property (such as sample mass) or to a spiked-in reference standard.
+
+#### Usage
+
+There are two ways to specify the information needed for performing instrument normalization on an nmrData object: 
+
+1. Specify `sample_property_cname`. This should be used when normalizing to a sample property, such as sample concentration or mass, is desired. Here, `sample_property_cname` gives the name of the column in `f_data` which contains the property to use for normalization. If any samples have a missing value for this column, and error is returned. 
+
+2. Specify `metabolite_name`. This should be used when normalization to a spiked in standard is desired. Here `metabolite_name` gives the name of the metabolite in e_data (and e_meta, if present) corresponding to the spiked in standard. If any samples have a missing value for this metabolite, an error is returned. 
+
+The option to backtransform the data after either of these NMR normalizations is offered, if the user would like to ensure values are on a scale similar to their raw values before normalization. The following values are calculated and/or applied for backtransformation purposes: 
+
+- If normalization using a metabolite in `e_data` is specified, the location parameter is the median of the values for  `metabolite_name` 
+
+- If normalization using a sample property in `f_data` is specified, the location parameter is the median of the values in `sample_property`
+
+We'll use the example NMR dataset from the `pmartRdata` package.
+
+```{r}
+mynmr <- data(nmr_object_identified)
```

Comment:
This block errors out for me, I think this should be changed to:

```
data(nmr_identified_object)
summary(nmr_identified_object)

# log2 transform the values
nmr_object <- edata_transform(omicsData = mynmr, data_scale = "log2")
```

Reply:
You're correct, good catch!